package.json

```diff
@@ -73,6 +73,7 @@
     "react-native-background-timer": "2.4.0",
     "react-native-callstats": "3.61.0",
     "react-native-default-preference": "1.4.2",
+    "react-native-device-info": "^8.0.2",
```

Comment:
There is no change in package-lock.json, but there should be changes because of this added dependency. Please run npm install and commit the changes